### PR TITLE
Add tutorial topic and update link on index.md

### PR DESCRIPTION
**Pull request by btbristow**
I expected to add a link on `index.md` with a `.md` suffix, but none of them use one. So I added mine with the same format as the others.

### CODE REVIEW

File: docs/tutorials/update-task-description.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+# Update a task description
+
+In this tutorial, you will learn how to use Postman to update a task description. This use case is common, since your users will need to to add task details as their deadlines approach. 
+
+This tutorial takes about 15 minutes to complete. 
+
+## Before you begin
+
+Before starting this tutorial, install Postman and json-server. Then, sync the To-Do service to your  GitHub repository so that you can clone it to your desktop. For a complete set of steps, see Before you start a tutorial.
```
Comment (rbwatson) [inline docs/tutorials/update-task-description.md]: Could you refer to the [Before you start a tutorial](https://uwc2-apidoc.github.io/to-do-service-public/tutorials/before-you-start-a-tutorial.html) topic here?

File: docs/tutorials/update-task-description.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    ```shell
+    cd path/to/<your-github-workspace>/to-do-service/api
+    # Run the service against the database file
+    ./json-server.sh -w to-do-db-source.json
+    ```
+
+2. In Postman, add a new request with these values:
+   
+    * **METHOD** — `GET`
+    * **URL** — `{localhost}:{3000}/tasks`
```
Comment (rbwatson) [inline docs/tutorials/update-task-description.md]:
```suggestion
    * **URL** — `http://localhost:3000/tasks` 
```
If you're specifying the actual URL, wouldn't it look more like this?
Using the \{ and \} notation, would imply that those values are substitutes for the actual values used by the reader. Either way is fine, depending on the context.

File: docs/tutorials/update-task-description.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            "task_description": "My example description",
+            "id": "1"
+        },
+        {
+            "task_title": "My new task",
+            "task_description": "My example description",
+            "id": "2"
+        },
+    ]
+    ```
```
Comment (rbwatson) [inline docs/tutorials/update-task-description.md:50]: These examples don't match the documentation of the task resource: [task](https://uwc2-apidoc.github.io/to-do-service-public/api/task). Can you update them to match, please?

File: docs/tutorials/update-task-description.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+## Update the task description
+
+Now that you know the task `id`, send a `PATCH` request to the `/tasks/{id}` endpoint to update the description. 
+
```
Comment (rbwatson) [inline docs/tutorials/update-task-description.md:55]: I like the overview of the step.

File: docs/tutorials/update-task-description.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+    ```shell
+        cd path/to/<your-github-workspace>/to-do-service/api
+        # Run the service against the database file
+        ./json-server.sh -w to-do-db-source.json
+    ```
+
+2. In Postman, add a new request with these values:
+    * **METHOD** — `PATCH`
+    * **URL** — `{localhost}:{3000}/tasks/{id}`
```
Comment (rbwatson) [inline docs/tutorials/update-task-description.md]: See note above and match this to however you decide to go.

File: docs/tutorials/update-task-description.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+    ```json
+    {
+    "task_description": "New task description"
+    }
+    ```
+
+4. Click **Send**. 
+   The To-Do service updates the task description and returns a `200 OK`. 
+
```
Comment (rbwatson) [inline docs/tutorials/update-task-description.md:81]: ... and a response body with the new task resource, correct?

**Review by rbwatson — CHANGES_REQUESTED** [review]
This is a good tutorial.
It just has a couple of inconsistencies to sort out and then it'll be ready.

Thanks!

File: docs/tutorials/update-task-description.md (hunk 6 above)
Comment (btbristow) [reply docs/tutorials/update-task-description.md:81]: I have to assume yes, as some service that I can't locate is running on the port that json-server wants (and json-server seems to no longer allow me to change ports). I will fix that over the weekend, but submit this change now. Resolving.

**Review by rbwatson — APPROVED** [review]
Rebased and squashed for merge.
LGTM

**Pull request merged**